onnxruntime/test/python/onnx_backend_test_series.py

```diff
@@ -117,8 +122,8 @@ def create_backend_test(testname=None):
                                  '^test_gru_seq_length_cpu.*')
         global version_tag
         if version_tag == 'onnx141' or onnx.__version__ == '1.4.1':
```

Comment:
This script only works with the tipmaster. 
We guarantee onnxruntime can run the old models, but we don't guarantee onnxruntime package can work with old onnx package versions (like 1.2.0). As ONNX python API is changing, it's hard to achieve source code level backward compatibility.